buildscripts/agent/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -ex
+
+MAYA_AGENT_NETWORK=$1
+
+CONTAINER_IP_ADDR=$(ip -4 addr show scope global dev "${MAYA_AGENT_NETWORK}" | grep inet | awk '{print $2}' | cut -d / -f 1)
+
+# Start agent service
+exec /usr/local/bin/maya-agent up -bind="${CONTAINER_IP_ADDR}" 1>&2
```

Comment:
@kmova maya-agent up is not a command, it's maya-agent monitor.

Reply:
Why to bind when we can start it on localhost......and for communication purpose we will use k8s service